ansible/group_vars/tag_Environment_alpha

```diff
@@ -55,26 +58,17 @@ register_groups:
     - field
 
   multi:
-    - academy-school-eng
-    - diocese
     - government-organisation
     - government-service
     - internal-drainage-board
-    - la-maintained-school-eng
     - local-authority-eng
     - local-authority-sct
     - local-authority-type
     - notifiable-animal-disease
     - notifiable-animal-disease-confirmation-category
     - notifiable-animal-disease-investigation-category
-    - prison
-    - religious-character
+    - principal-local-authority
+    - prison-estate
     - registration-district
-    - school-admissions-policy
-    - school-authority-eng
     - school-eng
-    - school-gender
-    - school-phase
-    - school-tag
```

Comment:
They have remove those as fields completely from school-eng and also removing the registers. See https://github.com/openregister/registry-data/pull/80.

Also, I have not removed the school-* registers from `registers.tf` yet as it is likely that they might end up in alpha at some point.